fedimint-core/src/db/mem_impl.rs

```diff
@@ -265,6 +273,11 @@ mod tests {
         fedimint_core::db::verify_remove_by_prefix(database()).await;
     }
 
+    #[test_log::test(tokio::test)]
+    async fn test_expect_write_conflict() {
+        fedimint_core::db::expect_write_conflict(database()).await;
+    }
+
```

Comment:
Is this the only test we can add? We should compare to the rocksdb impl that has the full test suite.

Reply:
Still relevant ^

Reply:
yes, we have same tests as rocksdb now

except testing_desecending order but that method is not implemented by memdb